src/si/momentum.rs

```diff
@@ -0,0 +1,202 @@
+            "kilogram terameters per second";
+        @kilogram_gigameter_per_second: prefix!(giga); "kg*Gm/s", "kilogram gigameter per second",
+            "kilogram gigameters per second";
+        @kilogram_megameter_per_second: prefix!(mega); "kg*Mm/s", "kilogram megameter per second",
+            "kilogram megameters per second";
+        @kilogram_kilometer_per_second: prefix!(kilo); "kg*km/s", "kilogram kilometer per second",
+            "kilogram kilometers per second";
+        @kilogram_hectometer_per_second: prefix!(hecto); "kg*hm/s", "kilogram hectometer per second",
+            "kilogram hectometers per second";
+        @kilogram_decameter_per_second: prefix!(deca); "kg*dam/s", "kilogram decameter per second",
+            "kilogram decameters per second";
+        @kilogram_decimeter_per_second: prefix!(deci); "kg*dm/s", "kilogram decimeter per second",
+            "kilogram decimeters per second";
+        @kilogram_centimeter_per_second: prefix!(centi); "kg*cm/s", "kilogram centimeter per second",
+            "kilogram centimeters per second";
+        @kilogram_millimeter_per_second: prefix!(milli); "kg*mm/s", "kilogram millimeter per second",
+            "kilogram millimeters per second";
+        @kilogram_micrometer_per_second: prefix!(micro); "kg*µm/s", "kilogram micrometer per second",
+            "kilogram micrometers per second";
+        @kilogram_nanometer_per_second: prefix!(nano); "kg*nm/s", "kilogram nanometer per second",
+            "kilogram nanometers per second";
+        @kilogram_picometer_per_second: prefix!(pico); "kg*pm/s", "kilogram picometer per second",
+            "kilogram picometers per second";
+        @kilogram_femtometer_per_second: prefix!(femto); "kg*fm/s", "kilogram femtometer per second",
+            "kilogram femtometers per second";
+        @kilogram_attometer_per_second: prefix!(atto); "kg*am/s", "kilogram attometer per second",
+            "kilogram attometers per second";
+        @kilogram_zeptometer_per_second: prefix!(zepto); "kg*zm/s", "kilogram zeptometer per second",
+            "kilogram zeptometers per second";
+        @kilogram_yoctometer_per_second: prefix!(yocto); "kg*ym/s", "kilogram yoctometer per second",
+            "kilogram yoctometers per second";
+
+        @ton_meter_per_second: prefix!(mega) / prefix!(kilo); "t*m/s", "ton meter per second",
+            "ton meters per second";
+
+        @kilogram_meter_per_minute: 1.666_666_666_666_666_6_E-2; "kg*m/min", "kilogram meter per minute",
+            "kilogram meters per minute";
+        @kilogram_meter_per_hour: 2.777_777_777_777_777_7_E-4; "kg*m/h", "kilogram meter per hour",
+            "kilogram meters per hour";
+        @kilogram_meter_per_day: 1.157_407_407_407_407_4_E-5; "kg*m/d", "kilogram meter per day",
+            "kilogram meters per day";
+
+        // I'm sorry but "slug feet per second" is a pretty great unit name.
```

Comment:
Could you remove this comment. Slug feet per second is funny. Maybe include an aside in the commit message?

Reply:
np, done